guides/aws/cloud-practitioner/README.md

```diff
@@ -0,0 +1,107 @@
+
+* Define what the AWS Cloud is and the basic global infrastructure
+* Describe basic AWS Cloud architectural principles
+* Describe the AWS Cloud value proposition
+* Describe key services on the AWS platform and their common use cases
+* Describe basic security and compliance aspects of the AWS platform and the shared security model
+* Define the billing, account management, and pricing models
+* Identify sources of documentation or technical assistance
+* Describe basic/core characteristics of deploying and operating in the AWS Cloud
+
+### Study for your exam
+
+The key areas covered by the exam are:
+
+* **Domain 1:** Cloud Concepts (28%)
+  * What is meant by AWS Cloud and its value proposition?
+  * What are the aspects of AWS Cloud economics?
+  * What are the different aspects of cloud architecture design principles?
+
+* **Domain 2:** Security (24%)
+  * What is the AWS Shared Responsibility Model?
+  * What is meant by AWS Cloud security and compliance concepts?
+  * What are the AWS Identity & Access Management (IAM) capabilities?
+  * What are the resources for security support?
+
+* **Domain 3:** Technology (36%)
+  * What methods can be used to deploy and operate in the AWS Cloud?
+  * What is meant by the AWS global infrastructure?
+  * What are the core AWS services?
+  * What are the resources for technology support?
+
+* **Domain 4:** Billing and Pricing (12%)
+  * What are the differences between the different AWS pricing models?
+  * What are the various AWS account structures?
+  * What are the resources for billing and pricing support?
+
+#### Resources
+
+To answer the questions above, we have compiled a list of resources that we think are useful for the AWS Certified Cloud Practitioner exam. 
+
+It is important to note that there are different methodologies of studying. Hence, choose whichever resources you think would work best for you.
+
+##### AWS Whitepapers
+* [Overview of Amazon Web Services](https://d0.awsstatic.com/whitepapers/aws-overview.pdf)
+* [Architecting for the Cloud: AWS Best Practices](https://d1.awsstatic.com/whitepapers/AWS_Cloud_Best_Practices.pdf)
+* [How AWS Pricing Works](http://d1.awsstatic.com/whitepapers/aws_pricing_overview.pdf)
+* [Compare AWS Support Plans](https://aws.amazon.com/premiumsupport/plans)
+
+##### Courses/Videos
+* [AWS Cloud Practitioner Essentials (Second Edition)](https://www.aws.training/Details/Curriculum?id=27076)
+* [A Cloud Guru - AWS Certified Cloud Practitioner 2019](https://acloud.guru/learn/aws-certified-cloud-practitioner)
+* [LinuxAcademy - AWS Cloud Practitioner (CLF-C01)](https://linuxacademy.com/course/aws-cloud-practitioner)
+
+##### Practice Tests
+
+We found the following list of practice tests to be useful and recommend attempting all of them to gain a solid understanding of the AWS Certified Cloud Practitioner.
```

Comment:
I still think we should indicate which are full exams (a cloud guru, linux academy, whizlabs) and which are sample papers that do not fully replicate the examination experience.

Reply:
All of those mentioned provide full exams. The links to the pages gives viewers the option to choose from doing free tests or paid tests; the latter is always a full exam. 

As we are suggesting that everyone attempt all of the practice tests, I do not think it is necessary to state which are sample exams and which are full exams. We give the users the freedom to choose what they want to do rather than us driving them towards doing a particular practice exam.